lib/readline.js

```diff
@@ -47,9 +47,9 @@ function Interface(input, output, completer, terminal) {
   }
   historySize = historySize || kHistorySize;
 
-  completer = completer || function() { return []; };
+  completer = completer || undefined;
```

Comment:
This line can probably be removed, with the following line just being:

```javascript
if (completer && typeof complete !== 'function') {
  ...
}
```

Thoughts?